forch/forchestrator.py

```diff
@@ -113,6 +114,9 @@ def __init__(self, config):
         self._metrics = None
         self._varz_proxy = None
 
+        self._config_hash_retry = 0
+        self._config_hash_max_retry = os.getenv('_CONFIG_HASH_MAX_RETRY', _CONFIG_HASH_MAX_RETRY)
```

Comment:
why env variable rather than config param?

Reply:
I thought we should hide this implementation from the user, but I think we do need to expose it in Forch config in case the default value is not large enough.

Reply:
But hide-or-not-hide is not env-vs-config... they are the same w.r.t. if they exist or not.

Reply:
Got it. Added `config_hash_cooling_sec` within `EventClientConfig` in forch config.